avocado/plugins/run.py

```diff
@@ -152,8 +152,11 @@ def configure(self, parser):
                              help="Inject [path:]key:node values into the "
                              "final multiplex tree.")
 
-    def _activate(self, args):
-        # Extend default multiplex tree of --mux_inject values
+    @staticmethod
+    def _process_mux_inject(args):
```

Comment:
This is pretty much the same code and naming of what's present in `avocado/plugins/multiplex.py`. I'd suggest to rename there as well to keep it sane.